src/modules/users/users.service.ts

```diff
@@ -363,10 +366,23 @@ export class UsersService {
       user.uploadCardDataWithDataNet = updateUserDTO.uploadCardDataWithDataNet;
       user.uploadCardEvidenceWithDataNet =
         updateUserDTO.uploadCardEvidenceWithDataNet;
+      user.status = updateUserDTO.status;  
       user.updatedAt = new Date();
 
       await this.userRepository.save(user);
 
+      if (updateUserDTO.status === 'I') {
```

Comment:
can we use a constant fr this i?

Reply:
OF course. Ready

Reply:
![image](https://github.com/user-attachments/assets/43dd1300-186b-4e18-a491-8228a4dcb539)